servicepulse/health-check-notifications.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Health check notifications
+summary: Describes how to setup email notifications for failing ServiceControl health checks
+component: ServicePulse
+reviewed: 2021-05-05
+related:
+---
+
+Every ServiceControl instance performs periodic health checks and raises notifications in case any of them fail. By default, notifications get published as [integration events](/servicecontrol/contracts.md) but it's also possible to get them delivered as email messages.
```

Comment:
Is it worth putting minimum versions of ServiceControl and ServicePulse on this page as a note or something?

Reply:
done